src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonCellModel.swift

```diff
@@ -24,8 +25,16 @@ class HealthCertifiedPersonCellModel {
 		title = AppStrings.HealthCertificate.Overview.covidTitle
 		name = healthCertifiedPerson.name?.fullName
 
+		let initialCertificate: HealthCertificate
```

Comment:
To reduce the complexity here, would it make sense to move the logic from line 18 and this logic to HealthCertifiedPerson and combine it?

Reply:
Extracted some logic into static funds now to reduce the complexity